css/style.css

```diff
@@ -752,6 +753,104 @@ ul.social-link .icon {
   margin-right: 140px;
 }
 
+.project-modal {
+  background: #fff;
+  border: 1px solid hsl(223, 12%, 89%);
+  width: 90%;
+  max-width: 90%;
+  height: 98vh;
+  padding: 0 10px;
+  margin: 12px auto;
+}
+
+.modal-container {
+  position: fixed;
+  inset: 0 0;
+  background-color: rgba(0, 0, 0, 0.8);
```

Comment:
- [ ] Please increase the image and project description size so they occupy the empty model height from the bottom and please also make sure your design matches with the Figma design.

### Your design

![image](https://user-images.githubusercontent.com/83803414/168036831-b03bf200-fede-4a10-9e39-3f47f509c474.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/168036794-063a6a46-40e2-4167-ab65-5831b2811c98.png)